prow/crier/reporters/github/reporter.go

```diff
@@ -150,10 +150,18 @@ func (c *Client) Report(_ *logrus.Entry, pj *v1.ProwJob) ([]*v1.ProwJob, *reconc
 
 	// TODO(krzyzacy): ditch ReportTemplate, and we can drop reference to config.Getter
 	err := report.Report(c.gc, c.config().Plank.ReportTemplateForRepo(pj.Spec.Refs), *pj, c.config().GitHubReporter.JobTypesToReport)
-	if err != nil && strings.Contains(err.Error(), "This SHA and context has reached the maximum number of statuses") {
-		// This is completely unrecoverable, so just swallow the error to make sure we wont retry, even when crier gets restarted.
-		err = nil
+	if err != nil {
+		if strings.Contains(err.Error(), "This SHA and context has reached the maximum number of statuses") {
+			// This is completely unrecoverable, so just swallow the error to make sure we wont retry, even when crier gets restarted.
+			log.WithError(err).Debug("Encountered unrecoverable error")
```

Comment:
nit: If I saw this in the logs I'd think there is a critical error with Prow 😈 It would be nice to clarify that this just means we are skipping retries for this PJ.

Reply:
done